apps/apps.c

```diff
@@ -149,20 +149,30 @@ int ctx_set_ctlog_list_file(SSL_CTX *ctx, const char *path)
 
 #endif
 
-int dump_cert_text(BIO *out, X509 *x)
+static unsigned long nmflag = 0;
+static char nmflag_set = 0;
+
+int set_nameopt(const char *arg)
 {
-    char *p;
+  int ret = set_name_ex(&nmflag, arg);
+
+  if (ret) 
+    nmflag_set = 1;
+		
+	return ret;
```

Comment:
Sorry, code style wise, this indentation isn't acceptable.  We use 4 space indent, but what got me was the indentation if the return statement.